.github/workflows/docs/build-docs

```diff
@@ -39,9 +39,11 @@ def build_module_docs():
     mod_docs = MODULES_DIR / "docs"
     mod_build = mod_docs / "build"
     conf_copy = mod_docs / "conf.py"
-    logo_copy = mod_docs / "Quantinuum_logo.png"
+    logo_copy_black = mod_docs / "Quantinuum_logo_black.png"
+    logo_copy_white = mod_docs / "Quantinuum_logo_white.png"
     shutil.copy(DOCS_DIR / "conf.py", conf_copy)
-    shutil.copy(DOCS_DIR / "Quantinuum_logo.png", logo_copy)
+    shutil.copy(DOCS_DIR / "Quantinuum_logo_black.png", logo_copy_black)
+    shutil.copy(DOCS_DIR / "Quantinuum_logo_white.png", logo_copy_white)
     remove_dir(mod_build)
     index_rst = mod_docs / "index.rst"
     with open(mod_docs / "intro.txt", "r") as f:
```

Comment:
Do we need to add these additional lines that are in the other extensions here? I just noticed that the diff was not the same as the other PRs.


```
    logo_copy_black.unlink()
    logo_copy_white.unlink()
```

Reply:
Oh, good spot. The line was missing before so the search and replace did not pick it up. I have now added that again. I think it is better to have them.